src/body/to_bytes.rs

```diff
@@ -13,6 +13,28 @@ use super::HttpBody;
 /// Care needs to be taken if the remote is untrusted. The function doesn't implement any length
 /// checks and an malicious peer might make it consume arbitrary amounts of memory. Checking the
 /// `Content-Length` is a possibility, but it is not strictly mandated to be present.
+///
+/// Create an empty `Body` stream.
+///
+/// # Example
+///
+/// ```
+/// let response = client.request(req).await?;
+///
+/// const MAX_ALLOWED_RESPONSE_SIZE: u32 = 1024;
+///
+/// let response_content_length: u32 = response.headers()
+///     .get(hyper::header::CONTENT_LENGTH)
+///     .expect("Failed to extract the CONTENT_LENGTH")
+///     .to_str().expect("Failed to_str() the CONTENT_LENGTH")
+///     .parse::<u32>().expect("Failed to parse the CONTENT_LENGTH");
```

Comment:
It does seem like a good idea to remind people in the example to care about size, so thanks! It should be possible to more easily get the reported size from the body itself, after importing `hyper::body::HttpBody`. Then you can check `response.body().size_hint()`.

Reply:
Addressed with commit d98dbfc9

Reply:
Addressed with commit d98dbfc9